.github/workflows/ci.yml

```
@@ -85,6 +85,18 @@ jobs:
         bundle --jobs 3 --retry 3
     - name: Run tests
       run: bundle exec ruby -w $(bundle exec ruby -e 'print File.join Gem.bindir, %q(rake)') test:all
+    - name: Check default stylesheet
+      if: matrix.primary
+      run: |
+        cd stylesheets
+        npm i --silent --no-audit --no-audit --no-fund --no-package-lock --no-progress
+        npm run build
+        cd ..
```

you can use `--prefix` to avoid `cd stylesheets` and `cd ..`


```suggestion
        npm i --silent --no-audit --no-audit --no-fund --no-package-lock --no-progress --prefix stylesheets
        npm run build --prefix stylesheets
```

---

Another solution would be to run `npm i` and `npm run build` in a dedicated task with `working-directory` (and then check if there's any uncommitted changes in another task)